04_numeronym/sahidakhtar1/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, len(vals))
+	for i, v := range vals {
+		result[i] = createNumeronym(v)
+	}
+	return result
+}
+
+func createNumeronym(s string) string {
+	trimedString := strings.TrimSpace(s)
+	var numeronym string
+	//If the string contains space or \t or \n then it will return the same string
+	if strings.ContainsAny(trimedString, " & \t &\n") {
```

Comment:
If your intent is to only check for space, `\t` and `\n`, then this is incorrect, it will also match `&`.
Use `" \t\n"` instead.

Reply:
Done